src/Sprites/OP2BmpLoader.cpp

```diff
@@ -23,7 +23,7 @@ void OP2BmpLoader::ExtractImage(std::size_t index, const std::string& filenameOu
 	std::vector<uint8_t> pixelContainer(imageMeta.scanLineByteWidth * imageMeta.height);
 	(*pixels).Read(pixelContainer);
 
-	IndexedBmpWriter::WriteScanLineIncluded(filenameOut, artFile.GetBitCount(index), imageMeta.width, -1 * imageMeta.height, palette, pixelContainer);
+	IndexedBmpWriter::WritePitchIncluded(filenameOut, artFile.GetBitCount(index), imageMeta.width, -1 * imageMeta.height, palette, pixelContainer);
```

Comment:
Okay, how about WritePixelPaddingIncluded

Reply:
I've read it a few times, and it feels a bit off too. I think part of it is juxtaposing "Pixel" with "Padding" creates a sense the words form one unit, and so you're only talking about the padding to the pixel data, rather than both the pixels and the padding. The other part is putting "Included" at the end sounds weird grammatically, like their needs to be a comma in the name. I'm fine with breaking English grammar rules in programming when there is a logical or hierarchical ordering that makes sense, but I don't get that sense here. If you swapped word order to `WritePixelsIncludingPadding` I think it would read better. Or you can shorten it to `WritePixelsAndPadding`.